action.yml

```diff
@@ -59,13 +59,32 @@ runs:
       uses: ublue-os/remove-unwanted-software@v6
       if: ${{ inputs.maximize_build_space == 'true' }}
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+      with:
+        install: true
+        cache-binary: ${{ inputs.use_cache }}
```

Comment:
I see this `use_cache` input being used everywhere, but can't find it declared anywhere, or a mention in GH docs. Where's this coming from?

Reply:
This should be an input on our side. I added it in the `testing` branch. @gerblesh make sure to grab that

Reply:
_that's why you implement features in a feature branch and not in `testing`_

Reply:
should be resolved in https://github.com/blue-build/github-action/pull/29/commits/4ef558bb3ecf36b5ea618db182f1ce660ea90d6c